### ISSUE TITLE
sbteclipse does not create <classpathentry kind="src" ...>  for "it" and "e2e" test directories

### ISSUE THREAD

**Issue by Ceilican** [typesafehub/sbteclipse#324]
I have created special folders and sbt tasks for integration tests and end-to-end tests, following these instructions: https://orrsella.com/2014/09/24/integration-and-end-to-end-test-configurations-in-sbt-for-scala-java-projects/ .

I expected sbteclipse to automatically add

```
  <classpathentry kind="src" path="src/it/scala"/>
  <classpathentry kind="src" path="src/e2e/scala"/>
```

to `.classpath`. But it didn't. I had to add this information manually.

**Comment by scottcarey** [comment]
Yeah, I have had issues with that too.   We have `src/it` directories that have to be manually added to 'java buld path' every time the projects are generated.

**Comment by DavidGregory084** [comment]
From the [docs](https://github.com/typesafehub/sbteclipse/wiki/Using-sbteclipse):
```
configurations

This setting of type Set[Configuration] lets you control the configurations to take into account. It defaults to Set(Compile, Test).
```
I think you will need to add `IntegrationTest` and whatever your e2e configuration is called to this key
